genconvert/input/my_normalize_zawgyi_transliteration_rules.txt

```diff
@@ -0,0 +1,75 @@
+# Transliteration to convert Zawgyi text to a more consistent version,
```

Comment:
This file probably needs a copyright header.  Copy the one from my-t-my-d0-zawgyi.txt

Reply:
Done